### ISSUE TITLE
Docker image support.

### ISSUE THREAD

**Issue by ShivamB25** [aredden/flux-fp8-api#17]
please. i can also try making a image with nvida cuda as base layer

**Comment by 0xtempest** [comment]
I have a docker setup working with this, and I have optimized API changes as well, I'm planning to make a large PR soon once I fix all the LoRA issues.

**Comment by aredden** [comment]
Ah cool! @0xtempest  If it's a very large PR, could it be done in pieces? I would like to be able to test each change individually since for a very large PR it can be tough to figure out what is doing what. Also yeah, docker should be relatively simple to set up since you can just use a pytorch container and install the requirements.

**Comment by 0xtempest** [comment]
Yeah I can break it down in smaller PR's, probably will be early next week as there's a lot to clean up

**Comment by ShivamB25** [comment]
@0xtempest Could you please keep your fork (please add updates). and i would love to test around

**Comment by ShivamB25** [comment]
@0xtempest possible ? i would also like to try helping with lora's issues
